src/main/java/seedu/address/model/Statistics.java

```diff
@@ -23,33 +28,120 @@ public Statistics(ReadOnlyStatistics toBeCopied) {
         resetData(toBeCopied);
     }
 
+    /** Initialises empty DayData for past MAX_SIZE days */
+    public void init() {
+        LocalDate currDate = LocalDate.now();
+        for (int i = 0; i < MAX_SIZE; i++) {
+            LocalDate tempLocalDate = currDate.minusDays(i);
+            String tempLocalDateStr = tempLocalDate.toString();
+            Date tempDate = new Date(tempLocalDateStr);
```

Comment:
Can't LocalDate -> Date directly?

Reply:
I am using the formatter in Date to check validity, for consistency as well (check Date class in Model). Thoughts?